recipes/linux-audit/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "linux-audit" %}
+{% set version = "3.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://people.redhat.com/sgrubb/audit/audit-{{ version }}.tar.gz
+  sha256: fd9570444df1573a274ca8ba23590082298a083cfc0618138957f590e845bc78
+
+build:
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('linux-audit', max_pin='x.x') }}
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - libtool
+    - make
+  host:
+    - openldap
+
+test:
+  commands:
+    - test -f "${PREFIX}"/lib/libaudit.so
+    - auvirt --help
+    - '"${PREFIX}"/sbin/aureport -v'
+    - '"${PREFIX}"/sbin/ausearch -v'
```

Comment:
@conda-forge/core, do we ship packages in $PREFIX/sbin or just put everything into $PREFIX/bin?

Reply:
We only add `$PREFIX/bin` to `$PATH`.